config/app/vanity-redirects.json

```diff
@@ -0,0 +1,35 @@
+{
+  "/about-big/customer-service/fraud": "/contact#fraud",
+  "/about-big/customer-service/making-a-complaint": "/contact#complaints",
```

Comment:
One downside of this being a JSON file (not JS) means we lose the `${anchors.contactComplaints}` config item here (eg. there's a hard dependency between the anchor string used here and the ID in the Contact template). Worth moving this file to JS to allow that syntax? JSON means we can't even leave a comment here to explain that that string is significant. On the other hand we could add that comment in the Contact page template? Dunno.

Reply:
I'm OK with loosing the `${anchors.contactComplaints}` config as we're about to move the contact page into the CMS anyway so becomes moot at that point.

Good point about the comments though, will move it back into JS, but keep the simpler k/v format.